wrappers/IWearLogger/src/IWearLogger.cpp

```diff
@@ -141,6 +185,7 @@ class IWearLogger::impl
         virtualSphericalJointKinSensors;
 
     std::unordered_map<std::string, std::string> wearable2MatlabNameLookup;
+    std::unordered_map<std::string, std::unique_ptr<yarp::os::BufferedPort<yarp::os::Bottle>>> wearable2YarpPortLookup;
```

Comment:
I know @Yeshasvitvs is a big `Bottle` user, but I'm really allergic to it. Can't we use something more high-level like a yarp vector?

Reply:
Done only for you @diegoferigo 😀 https://github.com/robotology/wearables/pull/123/commits/afacfdf1c362f7bf87ef28f853f25037a6a3ca8a